src/BookList.js

```diff
@@ -9,15 +9,20 @@ const query = gql`
         id
         title
         author
+        numPages
       }
   }
 `;
 
 const useStyle = makeStyles((theme) =>
   createStyles({
-    title: {
-      marginRight: theme.spacing(1),
+    content: {
+      textOverflow: 'ellipsis',
+      overflow: 'hidden',
     },
+    space: {
```

Comment:
Please call this something more meaningful like: `bookRow` or just `row` in this context.

Reply:
Ok.
I change it to ׳bookRow׳